src/dsp_tools/commands/excel2json/properties.py

```diff
@@ -83,6 +86,19 @@ def excel2properties(
     return props, True
 
 
+def _check_for_deprecated_syntax(df: pd.DataFrame) -> None:
+    _check_for_deprecated_isSequenceOf(df)
```

Comment:
my idea is to leave `_check_for_deprecated_syntax()` in place when I remove the `isSequenceOf` definitively. Kind of as infrastructure for future deprecation warnings.

Reply:
I would even leave it in for a little longer and raise an error instead.